avm/ptn/deployment-script/import-image-to-acr/main.bicep

```diff
@@ -0,0 +1,255 @@
+metadata name = 'import-image-to-acr'
+metadata description = 'This modules deployes an image to an Azure Container Registry.'
+metadata owner = 'Azure/module-maintainers'
+
+@description('Required. The name of the deployment script resource.')
+param name string
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Enable/Disable usage telemetry for module.')
+param enableTelemetry bool = true
+
+// ============ //
+// Parameters   //
+// ============ //
```

Comment:
```suggestion
```

An odd location for this comment. I'd suggest to remove it 😉

Reply:
Found it in a "template" I used. Can be removed.